okio-nodefilesystem/build.gradle

```diff
@@ -0,0 +1,58 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
```

Comment:
Should probably be a `org.jetbrains.kotlin.js` module

Reply:
Left as MPP for now, only because I don’t know the syntax.

Reply:
Tracking at #914